src/syft/lib/__init__.py

```diff
@@ -142,20 +145,26 @@ def _load_lib(*, lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
             _regenerate_unions(lib_ast=lib_ast, client=client)
 
 
-def load(lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
+def load(
+    *libs: TypeUnion[TypeList, TypeTuple, TypeSet, str],
+    options: TypeDict[str, TypeAny] = {},
+) -> None:
     """
     Load and Update Node with given library module
 
     Args:
-        lib: name of library to load and update Node with
+        *libs: names of libraries to load and update Node with (can be variadic, tuple, list, set)
         options: external requirements for loading library successfully
     """
-    try:
-        _load_lib(lib=lib, options=options)
-    except VendorLibraryImportException as e:
-        critical(e)
-    except Exception as e:
-        critical(f"Unable to load package support for: {lib}. {e}")
+    if not isinstance(libs[0], str):
+        libs = tuple(libs[0])
+    for lib in libs:
+        try:
+            _load_lib(lib=str(lib), options=options)
```

Comment:
Q: Why is the need to convert to string?

Reply:
To ensure only string objects get passed.

Now checking using `isinstance` before calling `_load_lib`, to better handle error reporting.